src/components/LoginDialog.jsx

```diff
@@ -9,7 +9,7 @@ import StyleFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
 
 export default function LoginDialog() {
   const location = useLocation();
-  const { showLoginDialog } = location.state || { showLoginDialog: undefined };
```

Comment:
Otherwise there's a warning about passing `undefined` when `open` expects a boolean.

Reply:
Oh yeah I noticed that.